spec/live/article_spec.rb

```diff
@@ -25,5 +25,16 @@ def valid_attributes
     it "can be created" do
       expect(section_article).not_to be_nil
     end
+
+    describe "#search" do
+      before { section_article }
+
+      it "finds the article", :vcr do
+        actual = client.articles.search(query: "What")
+
+        expect(actual.count).to be > 0
+        expect(actual.last.title).to eq("What are these sections and articles doing here?")
+      end
+    end
```

Comment:
I am guessing this is how to write test for the search factory above. Should there also be tests for the cases like nameError or the missing search query as well? I am not sure how does Zendesk/ruby tackles unit tests tbh.